tests/upgrade/test_upgrade_migration.py

```
@@ -0,0 +1,219 @@
+            fixture_store=fixture_store,
+            plan=self.plan_resource,
+            target_namespace=target_namespace,
+        )
+
+    def test_check_vms(
+        self,
+        prepared_plan: dict[str, Any],
+        source_provider: BaseProvider,
+        destination_provider: BaseProvider,
+        source_provider_data: dict[str, Any],
+        target_namespace: str,
```

Ruff (0.15.12)

[warning] Unused method argument: `target_namespace`

(ARG002)